firmware/src/caseTempController/fan.cpp

```diff
@@ -0,0 +1,29 @@
+//*****************************************//
+// includes                                //
+//*****************************************//
+#include "caseTempController/fan.h"
+
+
+//*****************************************//
+// defines                                 //
+//*****************************************//
+#define FREQUENCY   25000   // the PWM frequency (fan runs best at 25kHz)
+#define RESOLUTION  8       // the duty cycle resolution in bits (8 means 255 steps)
```

Comment:
how about expanding the existing fan stuff and adding another func to control the utility fan?

Reply:
I don't want to rip apart the current structure where everything concerning the controller is in one directory, so I'm kind of not feeling good with this. If you insist I will change it, though.